server/routes/termdb.singlecellSamples.ts

```diff
@@ -184,6 +183,8 @@ function validateDataNative(D: SingleCellDataNative, ds: any) {
 }
 
 function validateGeneExpressionNative(G: SingleCellGeneExpressionNative) {
+	G.singleCellGeneExpression2bins = {} // cache for binning gene expression values
```

Comment:
no need to repeat `singleCell` in the name. suggest to rename to `sample2gene2expressionBin`